methods/README.md

```diff
@@ -0,0 +1,12 @@
+## zkVM Methods
+
+This folder contains the [zkVM] portion of your [Bonsai] application. 
+The zkVM operation is organized into `host` and `guest`. 
+
+The `host` code can be found inside `methods/src/`, and the `guest` code is inside `guest/src/bin`. 
+
+To learn more about the architecture of Bonsai apps and zkVM apps, check out the [developer documentation]. 
+
```

Comment:
In the Bonsai Foundry template context, the host is managed in the Bonsai Relay (and debatably the app-specific contracts); so it either doesn't show up in this code at all or shows up in the `contracts` section.

Maybe revise to something like this? (although maybe this makes things more confusing, not less?)
```suggestion
In typical use cases, you will only need to edit the code inside `guest/src/bin` which is where you write the source code for your [guest program].

To learn more about the architecture of Bonsai apps and zkVM apps, check out the [developer documentation]. Note that the Bonsai Ethereum Relay acts as the zkVM host and so you do not need to write zkVM host code yourself.

[guest program]: https://dev.risczero.com/terminology#guest-program
```